### PR TITLE
feat: sexify the scrollbar

### PR DESCRIPTION
**Pull request by NiceAesth**
meme

styling scrollbars sucks now they deprecated `overflow: overlay`

there is also the option of using https://github.com/Grsmto/simplebar/tree/master/packages/simplebar-react up to u guys

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+::-webkit-scrollbar {
+  width: 17.5px;
+  background: transparent;
+}
+
+::-webkit-scrollbar-thumb {
+  border-radius: 10px;
+  border: 5px solid transparent;
+  background-clip: padding-box;
+  background-color: rgba(75, 85, 99, 0.5);
```
Comment (NiceAesth) [inline src/App.css:14]: i picked whatever for the color

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+body {
+  padding-left: calc(100vw - 100%);
```
Comment (NiceAesth) [inline src/App.css:2]: always preserve space for the gutter (100vw - 100% returns scrollbar size)

**Comment by tsunyoku** [comment]
what does it look like

**Comment by NiceAesth** [comment]
Fk u could have asked me at coe but uhhh look at https://my.flagin.bio

**Comment by cmyui** [comment]
Going to close for a lil bit & revisit approaches for all this kinda meta-level website shortly -- ty for the submission! might end up in master/probably with some more edits

**Pull request closed without merging**